docs/index.md

```diff
@@ -58,152 +64,79 @@ Compose has commands for managing the whole lifecycle of your application:
 - [Compose environment variables](env.md)
 - [Compose command line completion](completion.md)
 
-## Quick start
-
-Let's get started with a walkthrough of getting a simple Python web app running
-on Compose. It assumes a little knowledge of Python, but the concepts
-demonstrated here should be understandable even if you're not familiar with
-Python.
-
-### Installation and set-up
-
-First, [install Docker and Compose](install.md).
-
-Next, you'll want to make a directory for the project:
-
-    $ mkdir composetest
-    $ cd composetest
-
-Inside this directory, create `app.py`, a simple Python web app that uses the Flask
-framework and increments a value in Redis. Don't worry if you don't have Redis installed, docker is going to take care of that for you when we [define services](#define-services):
-
-    from flask import Flask
-    from redis import Redis
-
-    app = Flask(__name__)
-    redis = Redis(host='redis', port=6379)
-
-    @app.route('/')
-    def hello():
-        redis.incr('hits')
-        return 'Hello World! I have been seen %s times.' % redis.get('hits')
-
-    if __name__ == "__main__":
-        app.run(host="0.0.0.0", debug=True)
-
-Next, define the Python dependencies in a file called `requirements.txt`:
-
-    flask
-    redis
-
-### Create a Docker image
-
-Now, create a Docker image containing all of your app's dependencies. You
-specify how to build the image using a file called
-[`Dockerfile`](http://docs.docker.com/reference/builder/):
-
-    FROM python:2.7
-    ADD . /code
-    WORKDIR /code
-    RUN pip install -r requirements.txt
-    CMD python app.py
-
-This tells Docker to:
+## Features
 
-* Build an image starting with the Python 2.7 image.
-* Add the current directory `.` into the path `/code` in the image.
-* Set the working directory to `/code`.
-* Install the Python dependencies.
-* Set the default command for the container to `python app.py`
+#### Preserve volume data
 
-For more information on how to write Dockerfiles, see the [Docker user guide](https://docs.docker.com/userguide/dockerimages/#building-an-image-from-a-dockerfile) and the [Dockerfile reference](http://docs.docker.com/reference/builder/).
+Compose preserves all volumes used by your services. When `docker-compose up`
+runs, if it finds any containers from previous runs, it copies the volumes from
+the old container to the new container. This process ensures that any data
+you've created in volumes isn't lost.
 
-You can build the image by running `docker build -t web .`.
 
-### Define services
+#### Only recreate containers that have changed
 
-Next, define a set of services using `docker-compose.yml`:
+Compose keeps track of the configuration used to create a container. If you
+stop and start a service that hasn't changed its configuration Compose will
+re-use the old containers. In larger environments (with many containers) this
```

Comment:
Mary taught me that future tense should be avoided in tech writing; also "the old containers" is a bit awkward, maybe "the existing containers"? 

"In larger environments (with many containers)" is a bit redundant; I think this saves time in *any* environment, even small ones.

```Markdown
...If you restart a service that has no changes in its configuration, Compose
re-uses the existing container. Re-using the container allows you to make
changes to a service very quickly.
```

(Needs some tweaking, not entirely satisfied with that :smile:)